package.json

```diff
@@ -67,29 +67,29 @@
   "author": "nicolodavis@gmail.com",
   "license": "MIT",
   "devDependencies": {
+    "@babel/cli": "^7.2.3",
+    "@babel/core": "^7.2.2",
+    "@babel/plugin-proposal-class-properties": "^7.2.3",
+    "@babel/plugin-transform-async-to-generator": "^7.2.0",
+    "@babel/plugin-transform-modules-commonjs": "^7.2.0",
+    "@babel/preset-env": "^7.2.3",
     "@storybook/addon-actions": "^3.2.16",
     "@storybook/addon-knobs": "^3.3.10",
     "@storybook/addon-options": "^3.2.16",
-    "@storybook/react": "^3.4.11",
     "@storybook/addons": "^3.4.11",
-    "babel-cli": "^6.26.0",
-    "babel-core": "^6.26.3",
+    "@storybook/react": "^4.1.4",
+    "ajv": "^6.6.2",
```

Comment:
What is this dependency for?

Reply:
`ajv-keywords` wants it. Who uses that? I dunno man... dependency hell. I could take it or leave it, but it resolves this warning:

`npm WARN ajv-keywords@3.2.0 requires a peer of ajv@^6.0.0 but none is installed. You must install peer dependencies yourself.`